Jenkinsfile

```diff
@@ -589,6 +596,7 @@ pipeline {
                 sh script: '''
                 cd build
                 export BUILD_ABS_PATH=`pwd`
+                sudo ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
Why would this be necessary? This seems dangerous and will break random scripts on the target machine that assume `python` is `python2`.

also @lmwnshn to check later whether the benchmark machines have had `python` permanently modified, I remember it doesn't reset cleanly sometimes.

Reply:
Right, I am still figuring out how to run this correctly with Jenkins. I ran into a  `python` not found with the `shabang` here: http://jenkins.db.cs.cmu.edu:8080/blue/organizations/jenkins/testing-team%2Fterrier/detail/PR-31/13/pipeline